README.md

```diff
@@ -1,2 +1,3 @@
 # EatMod
 CPE101 Engineering Exploration Project
+asd
```

Comment:
What is this? Why do you commit and push this editing? LOL

Reply:
haha, That time I want to know that how commit and push work. So, I try to edit something that I think it's not affect to the other file.